features/guidance/roundabout.feature

```diff
@@ -707,3 +744,27 @@ Feature: Basic Roundabout
         When I route I should get
             | waypoints | route          | turns                                                                           |
             | a,h       | ab,ef,ef,fh,fh | depart,roundabout-exit-4,notification slight right,notification straight,arrive |
+
+
+     Scenario: Drive through roundabout
+        Given the node map
+           """
+              a
+            b e d  f
+              c
+            g   h
+           """
+
+        And the ways
+           | nodes | junction   | oneway |
+           | abcda | roundabout | yes    |
+           | edf   |            |        |
+           | gch   |            | yes    |
+
+        When I route I should get
+           | waypoints | bearings | route       | turns                           |
+           | e,f       | 90 90    | edf,edf,edf | depart,roundabout-exit-1,arrive |
```

Comment:
I don't think that the instruction should be suppressed, but with the PR it will be announced as "Enter the roundabout and take the 1st exit onto" that is not correct in my opinion and must be announced as a usual intersection, but it will require fix for counting.  Sure it is a topic for another PR after fixing #3979. I will add a note to check the test in a fix for #3979